examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -2153,6 +2190,78 @@ impl FrontendNode {
         Ok(())
     }
 
+    fn test_http_server_create(&mut self) -> TestResult {
+        // Create an HTTP server pseudo-Node.
+        let result = oak::http::init("[::]:8282");
+        expect_matches!(result, Ok(_));
+        let invocation_receiver = result.unwrap();
+        // Close the only read-handle for the invocation handle, which should
+        // trigger the HTTP server pseudo-Node to terminate (but we can't
+        // check that here).
```

Comment:
BTW, does a manual check (e.g. via the introspection server or via logging) confirm that the pseudo-Node terminates?